### DISCUSSION TITLE
Progress Bar Not Displaying When Navigating from Middle of Page

### DISCUSSION THREAD

**Discussion by madurapa** [livewire/livewire#8393] [7. Bugs (archived)] [answered]
### Livewire version

v3.4.12

### Laravel version

v10.48.10

### Which PHP version are you using?

PHP 8.3

### Steps To Reproduce

When you are at the top of the page, the progress bar displays correctly. However, if you scroll down and click on a navigated link in the middle of the page, the progress bar does not appear.

### Repository or Wirebox URL

N/A

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by donnebanget** [comment, marked as answer]
As an documentation, "When a page takes longer than 150ms to load, Livewire will show a progress bar at the top of the page."

in my case, i have navbar with fixed position and when scroll will hide some part of navbar. 

i'm add this on app.css

```css
#nprogress .bar {
    position: fixed !important;
    z-index: 9999999;
}
```

**Reply by madurapa** [reply]
Thanks a lot, mate! It worked like a charm.

I had added this CSS before, but I guess the value I used for the z-index wasn't high enough.

I also added the below for mobile, as I thought the bar wasn't visible enough on mobile devices.

```css
#nprogress .bar {
    height: 4px !important;
}
```
